locksmith/src/app.js

```diff
@@ -29,6 +36,12 @@ app.post(
   /^\/lock$/i,
   signatureValidationMiddleware.generateProcessor(lockSignatureConfiguration)
 )
+
+app.post(
+  /^\/events$/i,
```

Comment:
Just want to make sure that we add a test to ensure that this middleware gets triggered when a event is created?

Reply:
I'll add a negative test case to indicate that it is being triggered, similar to what we currently have with other paths that require signatures. 

In another PR i'll address the regexes in the middleware to ensure that callers don't fall through with missing/invalid signatures